retworkx/__init__.py

```diff
@@ -144,6 +146,9 @@ def distance_matrix(graph, parallel_threshold=300):
     :param bool as_undirected: If set to ``True`` the input directed graph
         will be treat as if each edge was bidirectional/undirected in the
         output distance matrix.
+    :param float null_value: An optional float that will treated as a null
+        value. If any element in the input matrix is this value it will be
+        treated as not an edge. By default this is ``0.0``.
```

Comment:
This does clarify one of my prev questions 👍

Reply:
Oh, this is a copy and paste error from #409 there is no input matrix for this function, it generates a matrix :)

Reply:
Done in: https://github.com/Qiskit/retworkx/pull/410/commits/ad32ec5fd08d00058b7a33e7771ab075730b33d8